src/tmpfiles/tmpfiles.c

```diff
@@ -196,7 +196,11 @@ typedef enum {
         _CREATION_MODE_INVALID = -EINVAL,
 } CreationMode;
 
-static bool arg_cat_config = false;
+static enum {
+        CAT_CONFIG_NO,
+        CAT_CONFIG,
+        CAT_CONFIG_TLDR,
+} arg_cat_config = CAT_CONFIG_NO;
```

Comment:
hmm, so you have a 2nd enum for cat flags that is not a set of flags though, but very close to it? this really really smells as if this should just be the same.

And I think CAT_CONFIG_NO should just also be a regular flag CAT_CONFIG_OFF, that works similar to JSON_FORMAT_OFF, and is generic.